docs/src/api/class-locatorassertions.md

```diff
@@ -1151,6 +1151,29 @@ Expected attribute value.
 ### option: LocatorAssertions.toHaveAttribute.timeout = %%-csharp-java-python-assertions-timeout-%%
 * since: v1.18
 
+## async method: LocatorAssertions.toHaveAttribute#2
```

Comment:
Shouldn't we rename `toHaveAttribute` to `toHaveAttribute#1`? I thought that tooling would complain!

Reply:
Tried rolling the docs too - everything works fine.